computation/src/test/java/com/powsybl/computation/CompletableFutureTaskTest.java

```diff
@@ -40,10 +40,39 @@ private static List<Executor> executors() {
                 Executors.newSingleThreadExecutor(),
                 Executors.newCachedThreadPool(),
                 Executors.newWorkStealingPool(),
+                new MyTestExecutorWithException(),
                 ForkJoinPool.commonPool()
         );
     }
 
+    // Very basic, spawn a new thread
+    // and allow to wait for the end of the command
+    // just keep exception to be able to assert,
+    // you should only it to launch one command
+    // because it has just one latch and one exception
```

Comment:
```suggestion
    // Very basic executor that spawns a new thread
    // and allows to wait for the end of the command.
    // It just keeps an exception to be able to assert it.
    // You should use it to launch only one command
    // because it has just one latch and one exception
```